www/templates/user-vhost-warning

```diff
@@ -1,23 +1,25 @@
-Dear {group}:
+Hi {group},
 
-The OCF staff noticed that your group's website [1] does not have either
-the "Hosted by OCF" banner [2] or University disclaimer [3] required by OCF
-and university virtual hosting rules. All student group websites are
-required to display the "Hosted by OCF" banner as well as the student group
-disclaimer. Unfortunately, if you do not add both of them, we may be forced
-to temporarily disable your website to comply with the OCF's and
-university's requirements. Please add both sometime within the next 2 weeks
-to prevent disabling of your website.
+The staff at the OCF noticed that your group's website [1] may be missing a
+"Hosted by OCF" banner [2] or the University-mandated "student group
+disclaimer" [3]. All student group websites must display both elements to
+comply with OCF virtual hosting policy and University trademark
+regulation. If your website isn't fully compliant, we will have to
```

Comment:
don't think trademarks have anything to do with the university's disclaimer; when we last met with them, they were worried about content being seen as endorsed by the university. the disclaimer and the trademark issue ("Cal") are sort of orthogonal.

Reply:
The trademark is "Berkeley", as in berkeley.edu. But trademark compliance is naturally part of their policy for giving out subdomains, too, so it might work to refer to either set of requirements.

Reply:
i don't have a trademark on "ckuehl.me", but i still wouldn't let people have subdomains on it because it would look like i endorsed them. i think that's their primary concern (wrt the disclaimer), not people misusing trademarks.

Reply:
I have to say I think the concern is the same in both cases, whether they use law (trademark) or technology (zone ownership) to enforce it.

Reply:
the concern would still exist if they didn't hold bs trademarks on "berkeley" or "cal", though, which is why i suggest just saying "to comply with OCF and University policies" or something

Reply:
Alright, I can accept that conclusion.